### PR TITLE
feat: allow setting a size limit when importing SBOMs

### PR DESCRIPTION
**Pull request by ctron**
Also, switch the UI from tag to branch

### CODE REVIEW

File: common/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ edition = "2021"
 trustify-migration = { workspace = true }
 
 anyhow = { workspace = true }
+bytesize = { workspace = true, features = ["serde"] }
```
[the remark this answers is not in the document]
Comment (ctron) [reply common/Cargo.toml:10]: Yes, true.

File: common/Cargo.toml [hunk 2]
```diff
@@ -23,6 +24,7 @@ postgresql_embedded = { workspace = true, features = ["blocking", "tokio"] }
 regex = { workspace = true }
 reqwest = { workspace = true, features = ["native-tls"] }
 ring = { workspace = true }
+schemars = { workspace = true }
```
Comment (JimFuller-RedHat) [inline common/Cargo.toml:27]: cool, didnt know about this handy cargo

**Review by JimFuller-RedHat — APPROVED** [review]
LGTM

File: xtask/src/dataset.rs [hunk 3]
```diff
@@ -52,6 +52,7 @@ impl GenerateDump {
                         keys: vec!["https://access.redhat.com/security/data/97f5eac4.txt#77E79ABE93673533ED09EBE2DCE3823597F5EAC4".parse()?],
                         v3_signatures: true,
                         only_patterns: vec![],
+                        size_limit: None,
```
[the remark this answers is not in the document]
Comment (ctron) [reply xtask/src/dataset.rs:55]: True,  maybe I'll add that right away.

**Pull request merged**